3.10/aql/functions-geo.md

```diff
@@ -45,14 +45,40 @@ FOR doc IN doc // e.g. documents returned by a traversal
 `GEO_CONTAINS(geoJsonA, geoJsonB) → bool`
 
 Checks whether the [GeoJSON object](../indexing-geo.html#geojson) `geoJsonA`
-fully contains `geoJsonB` (Every point in B is also in A). The object `geoJsonA` has to be of type 
-`Polygon` or `MultiPolygon`, other types are not supported because containment is ill defined. 
-This function can be **optimized** by a S2 based [geospatial index](../indexing-geo.html).
+fully contains `geoJsonB` (every point in B is also in A). The object `geoJsonA`
+has to be of type `Polygon` or `MultiPolygon`, other types are not supported
+because containment is ill-defined.
```

Comment:
I'd rephrase this. 
Sth along this lines:
"The object 'geoJsonA' must conform to the currently supported types: 'Polygon' or 'MultiPolygon'. Support for other types will be introduced on future versions.

Reply:
We should not promise the impossible. Containment of points in lines is not numerically stable (which was meant by this "ill-defined" word. All other GeoJSON objects are zero- or one-dimensional and run into these numerical problems w.r.t. containment. Therefore we would not be wise to promise this. I will change the wording to explain this.